src/overlays/actors/ovl_Obj_Lift/z_obj_lift.c

```diff
@@ -30,29 +32,181 @@ const ActorInit Obj_Lift_InitVars = {
     (ActorFunc)ObjLift_Update,
     (ActorFunc)ObjLift_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B960D0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B960D8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B96160.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/ObjLift_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/ObjLift_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B9651C.s")
+s16 D_80B969C0[] = { 0x0000, 0x000A, 0x0014, 0x001E, 0x0028, 0x0032, 0x003C, 0x0000 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Lift/func_80B96560.s")
+typedef struct {
+    s16 x;
+    s16 z;
+} unkstruct_80B969D0;
```

Comment:
```suggestion
typedef struct {
    /* 0x00 */ s16 x;
    /* 0x02 */ s16 z;
} ObjLiftFramgentScale; // size = 0x4
```
can take or leave the naming suggestions again. sFramgentScales works for the array name too if you want